openerp/service/model.py

```diff
@@ -113,7 +113,8 @@ def _(src):
         tries = 0
         while True:
             try:
-                if openerp.registry(dbname)._init and not openerp.tools.config['test_enable']:
+                test_mode = openerp.tools.config['test_enable'] or openerp.tools.config['test_file']
```

Comment:
Should this be marked as OpenUpgrade specific change with a comment or something?